packages/dev/loaders/src/glTF/2.0/Extensions/EXT_lights_ies.ts

```diff
@@ -101,6 +102,11 @@ export class EXT_lights_ies implements IGLTFLoaderExtension {
 
                 this._loader._babylonLights.push(babylonSpotLight);
 
+                // Making sure we enable enough lights to have all lights together
+                for (const material of this._loader.babylonScene.materials) {
+                    (material as PBRMaterial).maxSimultaneousLights = Math.max((material as PBRMaterial).maxSimultaneousLights, this._loader.babylonScene.lights.length);
+                }

```

Comment:
It's not guaranteed that the lights are PBRMaterial. Is it okay to always set this property?

Reply:
Also, you can just put one of these somewhere near here instead of in multiple places. This is the end of the load.

https://github.com/BabylonJS/Babylon.js/blob/c10f1837eb3029b6ece47c2512a8c7791aef1df8/packages/dev/loaders/src/glTF/2.0/glTFLoader.ts#L458

Reply:
even better!